reduxApp2/dev/js/reducers/reducer-users.js

```diff
@@ -0,0 +1,54 @@
+export default function (state = {}, action) {
+
+  const validateEmail = (email) => {
+    var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    return re.test(email);
+  }
+
+
+  const validatePhoneNumber = (inputtxt) => {
+    var pnumber = /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/;
+    if (inputtxt.match(pnumber)) return true;
```

Comment:
@VARTIKASINGH1234 Why you're using if and else. Please convert it to one line statement.

Reply:
@VARTIKASINGH1234 Please read about double-negation (!!) operator in JavaScript and implement this to remove the if else block.

Reply:
I have removed if else statement using double negotiation operator